### PR TITLE
feat. add example in docs for removing default blocks

### LINKED ISSUE #897: Disable options
**Is your feature request related to a problem? Please describe.**
I don't want the user to upload audio.

**Describe the solution you'd like**
Toggle options like enable-disable audio or file uploads

### PR DESCRIPTION
**Pull request by hunxjunedo**
/claim #897 
fixes #897 

[Screencast from 03-07-2024 15:18:56.webm](https://github.com/TypeCellOS/BlockNote/assets/89797440/f37df774-4859-44a4-95f3-cd3604566b32)

### CODE REVIEW

**Comment by hunxjunedo** [comment]
check the preview. Audio and image are disabled:  https://blocknote-nje928jsg-typecell.vercel.app/basic/removing-block-option

File: examples/01-basic/11-removing-block-option/App.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    if(!unwantedBlocks.includes(blockName)){
+
+      //only allow to to pass if the blockName is not unwanted
+      editedSchema[blockName] = blockSchema
+    }
+  })
+
+  //now create the schema
+  const schema = BlockNoteSchema.create({
+    blockSpecs: {
```
Comment (YousefED) [inline examples/01-basic/11-removing-block-option/App.tsx:25]: does the following actually work?
```
blockSpecs: {
...defaultBlockSpecs,
audio: undefined
}```

or alternatively:

```
{ audio, ...keepBlockSpecs } = defaultBlockSpecs

blockSpecs: keepBlockSpecs;
```

I'd prefer the first option, but both are a bit cleaner / simpler then manually looping the specs imo

File: examples/01-basic/11-removing-block-option/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Removing Default Blocks Options
+
+In this example, we pass a custom schema to the editor function, which retuurns an editor which has our custom schema. While create thi schema we remove the blocks which we don't want, by using simple logic.
+
+**Relevant Docs:**
+
+- [Editor Setup](/docs/editor-basics/setup)
```
Comment (YousefED) [inline examples/01-basic/11-removing-block-option/README.md:7]: also add link to custom schemas

File: examples/01-basic/11-removing-block-option/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Removing Default Blocks Options
```
Comment (YousefED) [inline examples/01-basic/11-removing-block-option/README.md:1]: Let's name it `removing-default-blocks` and title "Removing default (built-in) blocks".

Also, the paragraph below has some typos + maybe you can clarify which blocks you're removing

**Review by YousefED — COMMENTED** [review]
awesome, some small requests but looks good otherwise

File: examples/01-basic/11-removing-block-option/App.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import "@blocknote/core/fonts/inter.css";
```
Comment (YousefED) [inline examples/01-basic/11-removing-block-option/App.tsx:1]: I think these files still need to be removed and an `npm run build` is necessary

Comment (hunxjunedo) [reply examples/01-basic/11-removing-block-option/App.tsx:1]: So do you want me to remove the import of styles.css only or other files as well ?

Comment (YousefED) [reply examples/01-basic/11-removing-block-option/App.tsx:1]: No I think you renamed the example, but kept all the old files?

Comment (hunxjunedo) [reply examples/01-basic/11-removing-block-option/App.tsx:1]: Oh sorry, my bad, that's a git issue, I forgot to stage some changes before commiting.

**Comment by hunxjunedo** [comment]
@YousefED where are we headed with this ? I think it's ready to be merged.

**Review by YousefED — APPROVED** [review]

**Comment by hunxjunedo** [comment]
Hey! @YousefED is there anything left regarding this PR ?

**Pull request merged**